framework/include/media/media_types.h

```diff
@@ -0,0 +1,38 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __MEDIA_TYPES_H__
+#define __MEDIA_TYPES_H__
+
+#include <sys/types.h>
+#include <stdbool.h>
```

Comment:
The sys/types.h and stdbool.h are not used in this file. Do you add them because they are mandatory on every media file?

Reply:
This is useless now. I'll remove this.

Reply:
remove that.